code-challenges.test.js

```diff
@@ -37,15 +63,66 @@ const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
 
 // b) Create the function that makes the test pass.
 
+describe("mixedData", () => {
+      it("returns only the remainder of the numbers array when divided by 3.", () => {
+        expect(mixedData(hodgepodge1)).toEqual([2, 0, -1, 0]);
+        expect(mixedData(hodgepodge2)).toEqual([2, 1, -1]);
+      })
+    })
+
+//declared a function
+//takes in an array as a parameter
+// iterate over the array using .filter to remove anything that is not a number in the array
+// iterate over new array using .map 
+// use modulo to find the remainders of the numbers divided by 3
+
+// const mixedData = (array) => {
+//   return array.filter(element => {
+//     return typeof element === 'number'
+//   }).map(number => number % 3)
+// }
+
+const mixedData = (array) => {
+  let Arr = array.filter(element => {
+        return typeof element === 'number'
+      })
+      return Arr.map(number => {
+       return number % 3
+      })
+    }
+
+    // line 54 why do we need to put number in quotes? 
```

Comment:
typeof will return the data type of whatever it is evaluating, however, it will be a string since that is how we are able to show a collection of characters.  In other words, to display the word number as the data type, it has to be a string.